website/source/guides/operations/outage.html.markdown

```diff
@@ -128,7 +128,10 @@ periodic basis.
 The next step is to go to the
 [`-data-dir`](/docs/configuration/index.html#data_dir) of each Nomad
 server. Inside that directory, there will be a `raft/` sub-directory. We need to
-create a `raft/peers.json` file. It should look something like:
+create a `raft/peers.json` file. The syntax will vary if using raft protocol version 2 or raft protocol version 3. 
+
```

Comment:
How can you quickly find out if running raft protocol version 2 or version 3?

Reply:
i can't think of a quick way... it's probably in the raft log, but that's not human-readable. if it's not explicit in the config, then you have to know what the default was for your version of nomad.

Reply:
I'm not sure when raft 3 was introduced. I think it would be very helpful to have a statement like: 

>Raft 3 became the default in Nomad version X.X

Reply:
It looks like the consul raft version is separate from the nomad raft version

https://learn.hashicorp.com/consul/day-2-operations/autopilot

>In Consul 0.8 this setting defaults to 2; in Consul 1.0 it will default to 3. For more information, see the Version Upgrade section on Raft Protocol versions.

Reply:
Consul and Nomad do default to different raft versions.

Nomad defaults and supported versions may be found here: https://www.nomadproject.io/guides/upgrade/upgrade-specific.html#nomad-0-8-0

The text below details how to determine the raft version from the format of `peers.json`.

There is also a `peers.info` file in the same directory as `peers.json` that details the peers file format and versioning.

Reply:
The protocol version can be found by typing `nomad server members`